helm-chart-sources/pulsar/values.yaml

```diff
@@ -1576,10 +1576,10 @@ pulsarAdminConsole:
     type: LoadBalancer
     ports:
     - name: http
-      port: 80
+      port: 8080
```

Comment:
Since these endpoints are meant to be hit by a browser, we want to use the standard ports 80 and 443. Otherwise, users would need to configure the port when logging into the Pulsar Admin Console.

Reply:
I had the ingress taking care of this. I take it the intent was to open directly.